cmd/mount_unix.go

```diff
@@ -736,7 +736,13 @@ func installHandler(m meta.Meta, mp string, v *vfs.VFS, blob object.ObjectStorag
 				logger.Errorf("exit after receiving signal %s, but umount does not finish in 30 seconds, force exit", sig)
 				os.Exit(meta.UmountCode)
 			}()
-			go func() { _ = doUmount(mp, true) }()
+			go func() {
+				if err := doUmount(mp, true); err != nil {
+					logger.Warnf("Umount failed: %s", err)
+				}
+				time.Sleep(time.Second * 30)
+				os.Exit(meta.UmountCode) // Just in case the above watchdog thread hangs
```

Comment:
FlushAll may take long than 30 seconds, some data may lose if we exit before that.

Reply:
How about only os.Exit when `doUmount` fails?

`FlushAll` only flushes memory buffers. Those are requests not acked to client, I suppose it's OK to drop them under abnormal conditions. Juicefs only promises a `close-to-open` consistency.